default.nix

```diff
@@ -1,27 +1,27 @@
-self: super:
+self: pkgs:
```

Comment:
@srhb had encouraged me to use `self` for these calls, if I recall our conversation on IRC correctly.